src/store/ViewStore.js

```diff
@@ -10,9 +10,10 @@ export default class ViewStore {
         this.ViewsMenu = this.menu.connect(Menu);
         when(() => this.context, this.onContextReady);
     }
+
+    staticViews = observable.box(ViewStore).get();
```

Comment:
very interesting design choice... could you explain your thought process on this?